index.html

```diff
@@ -58,6 +59,213 @@ <h1 class="identity_text">I'm Simba, I'm glad to see you!</h1>
+        <li class="technology">
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+
+    <div class="project">
+      <img src="Snapshoot_Portfolio2.png" class="project_img" alt="project_image">
+      <h1 class="project_title">Tonic</h1>
+      <ul class="project_details">
+        <li class="list_items">
+          <p class="list_items_text">CANOPY</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">Back End Dev</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">2015</p>
+        </li>
+      </ul>
+      <p class="project_description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="technology_used">
+        <li class="technology">
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
+    </div>
+
+    <div class="project">
+      <img src="Snapshoot_Portfolio3.png" class="project_img" alt="project_image">
+      <h1 class="project_title">Multi-Post Stories</h1>
+      <ul class="project_details">
+        <li class="list_items">
+          <p class="list_items_text">CANOPY</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">Back End Dev</p>
+        </li>
+        <li class="list_items">
+          <img src="Counter.png" alt="counter" class="counter_class">
+        </li>
+        <li class="list_items">
+          <p class="list_items_text_other">2015</p>
+        </li>
+      </ul>
+      <p class="project_description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+      <ul class="technology_used">
+        <li class="technology">
+          <p class="technology_text">html</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">css</p>
+        </li>
+        <li class="technology">
+          <p class="technology_text">javascript</p>
+        </li>
+      </ul>
+      <button class="project_button" type="submit">See Project</button>
```

Comment:
- [ ] Please use the same margin as the one in the Figma tag
your version : 
![image](https://user-images.githubusercontent.com/74744787/159060183-70ebf25e-79a1-4617-b6d7-b59eb94d0fa5.png)

Figma design 
![image](https://user-images.githubusercontent.com/74744787/159060565-89a63254-74c1-4bca-a72e-566e99d5dde3.png)